frontend/src/types.ts

```diff
@@ -64,6 +65,59 @@ export interface Users {
   updatedAt: string;
 }
 
-export interface PropsFriendApproveCard extends Friend {
-  getUserFriendsById: () => void;
+export interface FormData {
+  firstName?: string;
+  lastName?: string;
+  email?: string;
+}
+
+export interface Message {
+  isError: boolean;
+  message: string;
+}
+
+export interface FriendEmail {
+  email: string;
+}
+
+export type FriendShipsReturns =
+  Database["public"]["Functions"]["get_user_friends"]["Returns"];
```

Comment:
I'm not sure whether the code works well, but can you try like the below?

```js
const type FriendShipsReturns = Database.public.Functions.get_user_friends.Returns
```

Reply:
Will refactor at other issue.